pool/pool.go

```diff
@@ -2323,18 +2323,6 @@ func waitFor(ctx context.Context, params *WaitParams, condition func(context.Con
 	}
 }
 
-// NetworkInfo requests information about the NeoFS network of which the remote server is a part.
-//
-// Main return value MUST NOT be processed on an erroneous return.
-func (p *Pool) NetworkInfo(ctx context.Context) (netmap.NetworkInfo, error) {
-	cp, err := p.connection()
-	if err != nil {
-		return netmap.NetworkInfo{}, err
-	}
-
-	return cp.networkInfo(ctx, prmNetworkInfo{})
-}
-
```

Comment:
There was a conflict in func name. The difference was in one parameter, which maybe set externally without any problems, because don't require any extra parameters.
Decided to remove it, to solve the conflict

Reply:
It's OK.